### PR TITLE
jpa_OneToOne - OneToOne Bidirectional mapping done.

### PR DESCRIPTION
**Pull request by rakeshyadav78**
@rakeshyadav93 Kindly review the code.

### CODE REVIEW

File: springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/BookService.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import com.rakesh.jpamap.dto.Response;
+import com.rakesh.jpamap.entity.Book;
+import com.rakesh.jpamap.repository.BookRepository;
+
+@Service
+public class BookService {
+	@Autowired
+	private BookRepository bookRepository;
+
+	public Book saveBook(Book book) {
```
Comment (rakeshyadav93) [inline springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/BookService.java:17]: add comment here.

File: springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/StudentService.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import com.rakesh.jpamap.entity.Student;
+import com.rakesh.jpamap.repository.StudenRepository;
+
+@Service
+public class StudentService {
+
+	@Autowired
+	private StudenRepository studenRepository;
+
+	public Response getAllStudent() {
```
Comment (rakeshyadav93) [inline springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/StudentService.java:19]: add comment here

File: springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/BookService.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+		return bookRepository.save(book);
+	}
+
+	public Response getAllBook() {
+		Response response = new Response();
```
Comment (rakeshyadav93) [inline springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/BookService.java:22]: add explanation here.

File: springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/StudentService.java (hunk 2 above)
Comment (rakeshyadav78) [reply springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/StudentService.java:23]: comment added.

File: springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/BookService.java (hunk 3 above)
Comment (rakeshyadav78) [reply springboot_jpa_map/src/main/java/com/rakesh/jpamap/service/BookService.java:26]: explaination added

**Pull request merged**